cnames_active.js

```diff
@@ -275,6 +275,7 @@ var cnames_active = {
   "autodocs": "bguiz.github.io/autodocs", // noCF? (don´t add this in a new PR)
   "automata": "cname.vercel-dns.com", // noCF
   "automic": "automicjs.github.io",
+  "automind": "https://theautomind.github.io/"
```

Comment:
```suggestion
  "automind": "theautomind.github.io",
```
As the CI error notes, you do not need the `https://` prefix, nor the `/` suffix. Your entry also needs a comma at the end of the line.